### PR TITLE
Add install script for powershell

### LINKED ISSUE #458: Powershell install script for Windows
### Feature request

Add an `install.ps1` file to the repo which can be used to install Nixpacks on Windows, the same way as `install.sh` works.

### Motivation

This would allow for Nixpacks integration in [`flyctl`](https://github.com/superfly/flyctl)

### Contribution

_No response_

### PR DESCRIPTION
**Pull request by Milo123459**
This PR adds an install script to be used in PowerShell

Fixes #458 

- [x] Docs are updated if needed

### CODE REVIEW

File: install.ps1 [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+$NixpacksInstall = $env:NIXPACKS_INSTALL
+$BinDir = if ($NixpacksInstall) {
+  "$NixpacksInstall\bin"
+} else {
+  "$Home\.nixpacks\bin"
+}
+
+$NixpacksZip = "$BinDir\nixpacks.zip"
+$NixpacksExe = "$BinDir\nixpacks.exe"
+$Target = 'x86_64-pc-windows-msvc'
```
Comment (devtomio) [inline install.ps1:23]: This should also check for `aarch64` and `i686` architectures.

**Review by devtomio — CHANGES_REQUESTED** [review]

File: install.ps1 (hunk 1 above)
Comment (Milo123459) [reply install.ps1:23]: Wasn't exactly sure how to match architecture + I was basing this off of Deno's, which doesn't build for those targets.

Comment (devtomio) [reply install.ps1:23]: I'll open a pr to your fork.

Comment (Milo123459) [reply install.ps1:23]: thank you!

**Review by coffee-cup — APPROVED** [review]

**Pull request merged**